core/index.ts

```diff
@@ -20,7 +14,21 @@ function globalSession() {
   }
   return (global as any)._DF_SESSION as Session;
 }
-export const session = globalSession();
+const session = globalSession();
 
-// Used by generated index.js file.
-export const compileStandaloneSqlxQuery = compilers.compileStandaloneSqlxQuery;
+const supportedFeatures = {
+  arrayBufferIPC: true
```

Comment:
nit: should probably follow google TS style guide (https://google.github.io/styleguide/tsguide.html#camel-case), i.e. `arrayBufferIpc`